system/HTTP/URI.php

```diff
@@ -259,6 +263,8 @@ public function __construct(?string $uri = null)
      * If $silent == true, then will not throw exceptions and will
      * attempt to continue gracefully.
      *
+     * Note: Method not in PSR-7
```

Comment:
Let's deprecate this. Conditional exceptions is a really terrible design. 
```suggestion
     * @deprecated Method not in PSR-7
```

Reply:
Done.